stories/theme.BIO.introduction_biodiversity/carousel_content.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "src":"https://www.youtube.com/embed/WPOTTd27yyg",
+        "title":"Ecological insights from three decades of animal movement tracking across a changing Arctic",
+        "caption":"The Arctic Animal Movement Archive (AAMA) is a new and growing collection of studies describing movements of animals in and near the Arctic. The AAMA includes millions of locations of thousands of animals over more than three decades, recorded by hundreds of scientists and institutions. By compiling these data, the AAMA supports public data discovery, preserves fundamental baseline data for the future, and facilitates efficient, collaborative data analysis. We have used the AAMA to document climatic influences on the migration phenology of golden eagles, geographic differences in the adaptive response of caribou reproductive phenology to climate change, species-specific changes in terrestrial mammal movement rates in response to increasing temperature, and the utility of animal-borne sensors as proxies for ambient air temperature. The AAMA is a living archive that can be used to uncover other such changes, investigate their causes and consequences, and recognize larger ecosystem changes taking place in the Arctic.\n\nThis visualization shows multiple years of AAMA data as if all of the data were from the same year. Several different groupings of animals are shown: marine mammals, raptors, seabirds, shorebirds, terrestrial mammals, and waterbirds.  Snow and sea ice are also shown for context as they correlate to animal movements.\n\nVisualizers: Greg Shirah (lead), Lori Perkins\n\nFor more information or to download this public domain video, go to https://svs.gsfc.nasa.gov/4877"
```

Comment:
They are definitely not! I couldn't figure out how to get the new lines in because you can't encode them directly in a JSON without breaking the string. I don't really mind TBH.